xml/art_sle_ha_install_quick.xml

```diff
@@ -788,6 +793,98 @@ softdog                16384  1</screen>
        <keycap function="control"/><keycap>C</keycap>
       </keycombo>.
    </para>
+   </sect2>
+
+   <sect2 xml:id="sec-ha-inst-quick-test-with-cluster-script">
+    <title>Testing with the &test.script; Script</title>
+    <para>
+     The command &cmd.test.script; is a tool to run standardized tests
+     for a cluster.
+     It triggers cluster failures and verify configuration to find
+     problems. Before you use your cluster in production, it is
+     recommended to use this script to make sure everything works as expected.
+    </para>
+    <para>
+     The script supports the following checks:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <formalpara>
+       <title>Environment Check <option>-e</option>/<option>--env-check</option></title>
+       <para>
+        Check if hostname is resolvable, time service, watchdog, and
```

Comment:
This is very much reduced to keywords - how about phrasing it in a complete sentence: 
Checks if all host names can be resolved, and if time synchronization, watchdog, and firewall settings are configured correctly.

Reply:
Ok, fixed.